activiti-cloud-service-common/activiti-cloud-services-swagger/src/main/java/springfox/documentation/spring/web/plugins/DocumentationPluginsManager.java

```diff
@@ -0,0 +1,189 @@
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package springfox.documentation.spring.web.plugins;
+
+import static com.google.common.collect.FluentIterable.from;
+import static com.google.common.collect.Lists.newArrayList;
+import static springfox.documentation.spring.web.plugins.DuplicateGroupsDetector.ensureNoDuplicateGroups;
+
+import com.google.common.base.Function;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.plugin.core.PluginRegistry;
+import org.springframework.stereotype.Component;
+import springfox.documentation.service.ApiDescription;
+import springfox.documentation.service.ApiListing;
+import springfox.documentation.service.Operation;
+import springfox.documentation.service.Parameter;
+import springfox.documentation.service.PathDecorator;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spi.schema.contexts.ModelContext;
+import springfox.documentation.spi.service.ApiListingBuilderPlugin;
+import springfox.documentation.spi.service.ApiListingScannerPlugin;
+import springfox.documentation.spi.service.DefaultsProviderPlugin;
+import springfox.documentation.spi.service.DocumentationPlugin;
+import springfox.documentation.spi.service.ExpandedParameterBuilderPlugin;
+import springfox.documentation.spi.service.OperationBuilderPlugin;
+import springfox.documentation.spi.service.OperationModelsProviderPlugin;
+import springfox.documentation.spi.service.ParameterBuilderPlugin;
+import springfox.documentation.spi.service.ResourceGroupingStrategy;
+import springfox.documentation.spi.service.contexts.ApiListingContext;
+import springfox.documentation.spi.service.contexts.DocumentationContext;
+import springfox.documentation.spi.service.contexts.DocumentationContextBuilder;
+import springfox.documentation.spi.service.contexts.OperationContext;
+import springfox.documentation.spi.service.contexts.ParameterContext;
+import springfox.documentation.spi.service.contexts.ParameterExpansionContext;
+import springfox.documentation.spi.service.contexts.PathContext;
+import springfox.documentation.spi.service.contexts.RequestMappingContext;
+import springfox.documentation.spring.web.SpringGroupingStrategy;
+import springfox.documentation.spring.web.scanners.ApiListingScanningContext;
+
+@Component
```

Comment:
Same here, why do we need this class here?

Reply:
same as above